docs/pages/upgrading/agent-managed-updates/bootstrap-ec2-agent.mdx

```diff
@@ -0,0 +1,333 @@
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Principal": { "Service": "ec2.amazonaws.com" },
+         "Action": "sts:AssumeRole"
+       }
+     ]
+   }
+   EOF
+   $ aws iam create-role \
+     --role-name EC2TeleportRole \
+     --assume-role-policy-document file://trust-policy.json
+   ```
+
+1. Attach a permissions policy that allows reading only your secret (least privilege):
+
+   ```code
+   $ ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+   $ cat > policy.json <<EOF
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Action": "secretsmanager:GetSecretValue",
+         "Resource": "arn:aws:secretsmanager:us-west-2:${ACCOUNT_ID}:``secret:teleport/join-token-*"
+       }
+     ]
+   }
+   EOF
+
+   $ aws iam put-role-policy \
+     --role-name EC2TeleportRole \
+     --policy-name TeleportSecretPolicy \
+     --policy-document file://policy.json
+   ```
+
+1. Create an EC2 instance profile to use the role:
+
+   ```code
+   $ aws iam create-instance-profile --instance-profile-name EC2TeleportInstanceProfile
+   $ aws iam add-role-to-instance-profile \
+     --instance-profile-name EC2TeleportInstanceProfile \
+     --role-name EC2TeleportRole
+   ```
+
+1. Create `cloud-init.yaml` configuration file with cloud init data.
+
+   ```yaml
+   #cloud-config
+   packages:
+   - curl
+   - awscli
+
+   runcmd:
+     # Install Teleport using your cluster's script and define the group by query parameter.
+     # Script persists your updater flags (group, proxy, etc.)
+     - curl "https://<Var name="example.teleport.sh:443" />/scripts/install.sh?group=<Var name="development" />" | sudo bash
+
+     # Write agent config (join token) before starting the service
+     - |
+       TOKEN=$(aws secretsmanager get-secret-value --secret-id teleport/join-token --query SecretString --output text --region us-west-2)
+       cat >/etc/teleport.yaml <<EOF
+       version: v3
+       teleport:
+         join_params:
+           method: token
+           token_name: $TOKEN
+         proxy_server: <Var name="example.teleport.sh:443" />
+       ssh_service:
+         enabled: true
+       auth_service:
+         enabled: false
+       proxy_service:
+         enabled: false
+       EOF
+
+     # Enable and start Teleport service.
+     - systemctl enable --now teleport
+   ```
+
+1. Launch the EC2 instance with the instance profile:
+
+   ```code
+   # Specify the AMI image ID for the instance and a security group that allows at least outbound traffic.
+   $ aws ec2 run-instances \
+     --image-id <Var name="ami-xxxx" /> \
+     --instance-type t3.micro \
+     --iam-instance-profile Name=EC2TeleportInstanceProfile \
+     --security-group-ids <Var name="sg-xxxx" /> \
+     --region us-west-2 \
+     --user-data file://cloud-init.yaml
+   ```
+
+## Verifying on the instance
+Check updater status and persisted flags:
```

Comment:
We should tell users that this command should be run on the EC2 instance and not their laptop.
```suggestion
Once the EC2 instance is started, SSH on it to check the updater status:
```